stl/inc/vector

```diff
@@ -1275,18 +1305,32 @@ private:
         _Change_array(_Newvec, _Size, _Newcapacity);
     }
 
-    void _Clear_and_reserve_geometric(const size_type _Newsize) {
+#if _ITERATOR_DEBUG_LEVEL != 0
+    inline void _Check_all_orphaned_locked() const noexcept {

```

Comment:
We conventionally avoid unnecessary `inline` keywords. When a member function is defined within its class definition, it's implicitly `inline`. This occurs in other functions being added in this PR - please check all occurrences. (The occurrences that are necessary are when a non-templated member function is defined outside its class definition, or when a function is being explicitly specialized and loses the templateness that allows it to be header-only.)

Reply:
Ah, I remember that I got ODR violations from the `_Orphan_all_locked` and `_Swap_proxy_and_iterators_locked` and sprinkled it around.

Thanks a lot for the clarification. Whenever I encounter ODR problems I run away screaming